docs/architecture.md

```diff
@@ -1,14 +1,102 @@
-### cozy-client architecture
+### Cozy-client Architecture
 
 This is an overview of the cozy-client architecture:
 
 ![Architecture schema](./architecture.png)
 
-The architecture schema can also be accessed [online here](https://whimsical.co/6AJnnS7v5ePRcuKZysPAF5).
-It's created on https://whimsical.co/ — if you need to edit it, ask for an invite from one of the projects maintainers.
+<!-- The architecture schema can also be accessed [online here](https://whimsical.co/6AJnnS7v5ePRcuKZysPAF5).
+It's created on https://whimsical.co/ — if you need to edit it, ask for an invite from one of the projects maintainers. -->
 
-### Missing parts
+To explain the architecture, we will see how a request for data flows
+through cozy-client and how finally the UI is updated.
 
-- Init
-- Associations
-- Mutations
+### Query defitions
+
+To query data, we create a *query definition*: an object describing documents to fetch.
```

Comment:
Not really a document description but more a description on *how* to fetch

Reply:
Mmh not how, but more what don't you think ?

Reply:
Maybe a mix of both actually ; sort, limit, skip are "how" while find, where, id are more "what". But using just "what" looks fine ;)